frontend/.env.example

```
@@ -0,0 +1,30 @@
+# 3 つすべてが一致していること。1 つでもずれると redirect_uri_mismatch で止まる。
+VITE_OIDC_REDIRECT_URI=http://localhost:5173/login/callback
+
+# ログアウトで発行者側のセッションも終わらせる先。
+# 空だと、Cookie を消しても発行者側にログイン済みが残り、同じ端末で入り直せてしまう。
+VITE_OIDC_END_SESSION_URI=http://zitadel.localhost:8081/oidc/v1/end_session
+
+# 要求する範囲（空白区切り）。
+# openid だけだと id_token に名前もメールも載らず、offline_access が無いと
+# 更新用のトークンがそもそも発行されない（アクセストークンが切れた瞬間に
+# 全員ログイン画面へ飛ぶ）。省略時もこの 4 つを既定にしてある。
+VITE_OIDC_SCOPE=openid profile email offline_access
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)